xml_inscriptions/transcribed/MA.Glouc.HCM.L.03.8.5.xml

```diff
@@ -91,7 +90,8 @@
                         <!--  *************************************  <handDesc>  ********************************* -->
                       <!--     Contains information about types of writing and means of inscription.  -->
                         <handDesc hands="1">
-                            <handNote ana="#impressed.inscribed.carved"> </handNote>
+                            <handNote ana="#impressed.inscribed">
```

Comment:
Is there a particular reason we are removing third tier of specificity in most of these?

Reply:
I think we're making the general menus vaguer and adding specificity to the corresponding description boxes. Also, from the Encoding Instructions: "The most common type of writing by far is “inscribed”,
which is sometimes described by reporting sources as “carved”, “chiseled” or “engraved.""